exercises/concept/elyses-analytic-enchantments/enchantments.spec.js

```diff
@@ -9,103 +9,50 @@ import {
   getFirstEvenCardPosition,
 } from './enchantments';
 
-/**
- * @template T the expected return type
- * @typedef {Array<[number[], number, T]>} TestSingleMatrix
- */
-
-/**
- * @template T the expected return type
- * @typedef {Array<[number[], T]>} TestAllMatrix
- **/
-
-describe('elyses analytic enchantments', () => {
-  describe('getCardPosition', () => {
-    /** @type {TestSingleMatrix<number>} */
-    const getCardPositionTestCases = [
-      [[1, 2, 3], 1, 0],
-      [[1, 2, 2], 2, 1],
-      [[1, 2, 3], 4, -1],
-    ];
-
-    getCardPositionTestCases.forEach(([array, item, expected]) => {
-      test(`getCardIndex([${array}], ${item})`, () => {
-        expect(getCardPosition(array, item)).toStrictEqual(expected);
-      });
-    });
+describe('getCardPosition', () => {
+  test('when it is the first card', () => {
+    expect(getCardPosition([1, 2, 3], 1)).toBe(0);
```

Comment:
The "-1" test case (card not found in the array) is missing here. I though I saw it in the last iteration. :thinking: